readme.md

```diff
@@ -2,4 +2,6 @@
 
 Pretty simple to start, will iterate as needed. Hosted on [sarahmhigley.com](https://sarahmhigley.com/).
 
+To leave feedback, open an issue or contact me on twitter at @codingchaos.

```

Comment:
Pancake

Reply:
I should indeed have more pancakes in my readme :)